Content.Client/Chat/UI/SpeechBubble.cs

```diff
@@ -15,6 +15,7 @@ public abstract class SpeechBubble : Control
     {
         [Dependency] private readonly IEyeManager _eyeManager = default!;
         [Dependency] private readonly IEntityManager _entityManager = default!;
+        [Dependency] protected readonly ISharedChatManager _chatManager = default!;
```

Comment:
i'd say this one needs updating to adapt usage of our protos >_> so maybe leave todo at least? SpeechType  enum is irritating for example...